feat: add documentation for new usage page (#1956)
buildkite/docs: pages/test_analytics/test_executions.md

# Test executions

Each [Buildkite plan](https://buildkite.com/pricing) has test execution inclusions, which vary depending on the plan type and the number of users in your organization.

> You can find the test execution count on each run page

Reviewer: The sentence needs to end with either a period or colon. To make it more parallel with the Pipelines page, perhaps:

```suggestion
You can find the test execution details for a run at the bottom of the run page, and your organization's [total usage](#usage-page) in Settings.
```

<%= image "test_executions.png", alt: "Test executions run page" %>

## Usage page

> The [usage page](https://buildkite.com/organizations/~/usage) is available on every Buildkite plan and shows a breakdown of job minutes and test executions for your organization.

Reviewer: As with the other page:

```suggestion
The [Usage page](https://buildkite.com/organizations/~/usage) is available on every Buildkite plan and shows a breakdown of job minutes and test executions for your organization.
```

> <%= image "usage_page.png", alt: "Usage summary page" %>

Reviewer: Should we remove this image?

> The [test executions usage page](https://buildkite.com/organizations/~/usage/test_executions) graphs the total executions over the organization's current billing period. It includes a breakdown of the current usage by suite as well as a CSV download of usage over the period.

Reviewer: As with the other page:

```suggestion
The [Test executions usage page](https://buildkite.com/organizations/~/usage/test_executions) graphs the total executions over the organization's current billing period. It includes a breakdown of the current usage by suite and a CSV download of usage over the period.
```

> <%= image "test_executions_usage_page.png", alt: "Test executions usage summary page" %>

Reviewer: Should we remove this image?

Author: looks fine without it ✅
